typescript/rest-express/src/index.ts

```diff
@@ -1,16 +1,25 @@
-import { PrismaClient } from '@prisma/client'
-import * as bodyParser from 'body-parser'
```

Comment:
nice! just a little reminder if we replace `bodyParser` here we can also remove the dep from `package.json`